### ISSUE TITLE
Further information regarding the project

### ISSUE THREAD

**Issue by Gondhi-Sahana** [antoprince001/QA_Master#2]
Did u consider any research paper for this project? If yes, please let me know.
How did you calculate the accuracy of this project?

**Comment by antoprince001** [comment]
For the implementation, I did not refer to any research paper but mostly the medium article referenced in the Readme section. But you can find the following links useful for the pretrained model used in this project 

- https://huggingface.co/bert-large-uncased-whole-word-masking-finetuned-squad  
- https://arxiv.org/pdf/1810.04805v2.pdf
- https://web.stanford.edu/class/archive/cs/cs224n/cs224n.1194/reports/default/15848021.pdf
- https://www.researchgate.net/publication/351095836_BERT-CoQAC_BERT-based_Conversational_Question_Answering_in_Context
